docs/topics/serialization.txt

```diff
@@ -241,13 +241,13 @@ JSON in the following way::
         }
     ]
 
-The formatting here is a bit simpler than with XML. The whole collection
-is just represented as an array and the objects are represented by JSON objects
-with three properties: "pk", "model" and "fields". "fields" is again an object
+The formatting here is less complex than with XML. The whole collection is
```

Comment:
i'd revert this one, "a bit simpler" and "less complex" are two ways of saying the same thing